README.md

```diff
@@ -104,6 +121,12 @@ kubectl apply -f https://openfunction.sh1a.qingstor.com/crds/v0.7.0/openfunction
     kubectl apply -f https://openfunction.sh1a.qingstor.com/crds/v0.7.0/tekton-pipelines.yaml
     ```
 
+#### Install new chart
```

Comment:
Simply installing the new chart will not upgrade all the CRDs?

Reply:
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations

https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md#upgrading-chart

CRDs can be managed in templates , keda and contour both manage CRDs this way, but then `helm uninstall` will delete all custom resources(for us is Function), dapr does not manage CRDs this way. Also Knative's webhook blocks the patch operation performed by helm upgrade ，so currently CRDs are not managed in this way.